docs/docker-env-setup.md

```diff
@@ -0,0 +1,95 @@
+# Nydus Setup for Docker(Moby) Environment
+## Install Docker Nydus Graph Driver for Docker
+1. For older versions of Docker(Moby) lower than v24, please use [Docker Nydus Graph Driver](https://github.com/nydusaccelerator/docker-nydus-graphdriver)
+
+## Install Nydus Snapshotter for Docker(Moby) with Systemd
+1. Docker(Moby) newer than [5c1d6c957b97321c8577e10ddbffe6e01981617a](https://github.com/moby/moby/commit/5c1d6c957b97321c8577e10ddbffe6e01981617a) is needed on your host. The commit is expected to be included in Docker v24. 
+```
+git clone https://github.com/moby/moby
+cd moby
+make binary
+cd bundles/binary
+sudo systemctl stop docker
+sudo systemctl stop containerd
+sudo cp ./* /usr/bin/
+```
```

Comment:
Can we use the official binaries so that users don't have to build from scratch.
```
curl -fsSL https://get.docker.com -o get-docker.sh
sudo CHANNEL=nightly sh get-docker.sh
```

Reply:
Thanks for the suggestion, I'll revise this part.

Reply:
ok... I just looked at the shell code and saw the `nightly` option. Let's stick to the DIY approach until an official release is available. E.g., we can use `channel=test` to install the latest -rc releases.

Reply:
OK, test channel can work normally, but now the next version of -rc release seems to have not yet been released, and the Nydus image cannot be supported for the time being. We could change it later.